src/components-examples/material/datepicker/datepicker-locale/datepicker-locale-example.html

```diff
@@ -1,8 +1,8 @@
 <mat-form-field appearance="fill">
+  <mat-hint>{{getDateFormatString()}}</mat-hint>
```

Comment:
I think the convention is to put the `mat-hint` after the input so the order matches the visual order.